docs/docs/guides/build/external-pipelines/javascript-pipeline.md

```diff
@@ -26,7 +30,7 @@ To run the examples, you'll need to install:
 - The following Python packages:
 
    ```bash
-   pip install dagster dagster-webserver
+   uv pip install dagster dagster-webserver
```

Comment:
I think this isn't necessary if you `uvx create-dagster project` and then source the resulting venv.

I would recommend the following for the prereq section:
-  Turn it into a section with an H2 heading (`## Prerequisites`)
- Remove the following lines:
  - "Familiarity with Assets"
  - "A basic understanding of JavaScript and Node.js"
  - The `pip install` line
- Add a line about needing to `uvx create-dagster project`, cd to the project, and source the venv